core/deployment/src/main/java/io/quarkus/deployment/ThreadLocalRandomProcessor.java

```diff
@@ -1,15 +0,0 @@
-package io.quarkus.deployment;
-
-import java.util.concurrent.ThreadLocalRandom;
-
-import io.quarkus.deployment.annotations.BuildStep;
-import io.quarkus.deployment.builditem.nativeimage.RuntimeReinitializedClassBuildItem;
-
-public class ThreadLocalRandomProcessor {
-    @BuildStep
-    RuntimeReinitializedClassBuildItem registerThreadLocalRandomReinitialize() {
-        // ThreadLocalRandom is bugged currently and doesn't reset the seeder
-        // See https://github.com/oracle/graal/issues/1614 for more details
```

Comment:
This was fixed in GraalVM `19.3.0`. I'm not sure it's OK to remove the Quarkus workaround now as the issue will be present for someone that uses Quarkus `1.1` with a GraalVM version lower than `19.3.0`. Maybe we should wait a few GraalVM versions before removing it.

Reply:
I would vote for removing it now. Otherwise we will forget.

I will add a prominent note that people should upgrade to 19.3.0.